src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,230 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    private static final String LICENSE_NUMBER_LITERAL = "license_number";
+    private static final String NAME_LITERAL = "name";
+    private static final String CAR_ID_LITERAL = "car_id";
+    private static final String CAR_MODEL_LITERAL = "model";
+    private static final String MANUFACTURER_ID_LITERAL = "manufacturer_id";
+    private static final String MANUFACTURER_NAME_LITERAL = "manufacturer_name";
+    private static final String MANUFACTURER_COUNTRY_LITERAL = "manufacturer_country";
+
+    @Override
+    public Car create(Car car) {
+        String insertRequest = "insert into cars (model, manufacturer_id) values (?, ?)";
+        try (Connection connection = ConnectionUtil.getConnection();
+                PreparedStatement createCarStatement = connection
+                        .prepareStatement(insertRequest, Statement.RETURN_GENERATED_KEYS)) {
+            createCarStatement.setString(1, car.getModel());
+            createCarStatement.setLong(2, car.getManufacturer().getId());
+            createCarStatement.executeUpdate();
+            ResultSet generatedKeys = createCarStatement.getGeneratedKeys();
+            while (generatedKeys.next()) {
+                car.setId(generatedKeys.getObject(1, Long.class));
+            }
+        } catch (SQLException throwables) {
+            throw new DataProcessingException("Can`t create car: " + car, throwables);
+        }
+        insertDrivers(car);
+        return car;
+    }
+
+    @Override
+    public Optional<Car> get(Long id) {
+        String getRequest = "SELECT cars.id As car_id, cars.model, "
+                + "manufacturers.name AS manufacturer_name, manufacturers.id AS manufacturer_id, "
+                + "manufacturers.country AS manufacturer_country, "
+                + "cars.is_deleted FROM cars JOIN manufacturers "
+                + "ON cars.manufacturer_id = manufacturers.id "
+                + "WHERE cars.id = ? AND cars.is_deleted = FALSE";
```

Comment:
NOTE:
The same.
Somethin in uppercase, something in lowercase, something like 'As'.